### ISSUE TITLE
heatOnly option

### ISSUE THREAD

**Issue by G0sch** [PJCzx/homebridge-thermostat#50]
Hey

Should the "heatOnly" Option remove the "COOL" option in Home App?
If yes it looks like its not working.

![image](https://user-images.githubusercontent.com/51472553/73338998-07a53e00-4278-11ea-8ec1-82d09d08fc89.png)
<removed apiroute URL>

![image](https://user-images.githubusercontent.com/51472553/73339033-1c81d180-4278-11ea-8efd-135b38baee39.png)

**Comment by PJCzx** [comment]
@clvz do you have any input for @G0sch ?I'm not using my own plugin anymore ^^

**Comment by clvz** [comment]
Hello @G0sch ,

This is strange. Here's my config, that works. It doesn't seem much different from yours:
```
{
       "accessory": "Thermostat",
       "name": "Thermostat Séjour",
       "apiroute": "http://127.0.0.1:5000/room/sejour",
       "currentHumidity": false,
       "heatOnly": true,
       "maxTemp": 22.0,
       "minTemp": 7.0,
       "model": "ThermoPy",
       "serial": "0123",
    }
```

On your side, have you tried:
1. restarting homebridge
2. killing / reopening the Homekit app on your iDevice so that the config update is taken into account?

**Comment by G0sch** [comment]
Hello @clvz 

Yes tried it several time. 
Even tried to re-install the plugin :-)

thx
g0sch

**Comment by G0sch** [comment]
@clvz 
I cloned the repo and added "validValues: [0,1]" - looks like it is working now :-)

**Comment by PJCzx** [comment]
@G0sch Great, can you do a pull request ? this will allow to provide your work to other people that could be interesed ;) Further more, I'll publish it on NPM for an easy update

**Comment by G0sch** [comment]
sorry for the late response - i'm not so familiar with GitHub :-/
Have to read how it works in detail

**Comment by PJCzx** [comment]
it's quite easy, You make a fork, then you make your change. You will be able to make a pull request from your repo to mine ;)

**Comment by G0sch** [comment]
done - could you check it, please :-)

**Comment by PJCzx** [comment]
Corrected in #51 by @G0sch thx !

**Comment by PJCzx** [comment]
Published on npm as `homebridge-thermostat@0.0.52`
